### PR TITLE
[TR] Merge master 1.3

### PR DESCRIPTION
**Pull request by solivier**
behat : http://ci.akeneo.com/job/pim-pr-behat/2438/console

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -480,6 +501,16 @@
 - PIM-3632: Correctly show scopable attribute icons on scope change
 - PIM-3583: Fix the bad parsed filter value with spaces
 
+# 1.2.x
```
Comment (solivier) [inline CHANGELOG.md]: wrong

File: src/Pim/Bundle/EnrichBundle/Twig/CategoryExtension.php [hunk 2]
```diff
@@ -319,34 +348,34 @@ protected function formatTree(
...
 
     /**
      * Format categories
      *
-     * @param array  $categories
-     * @param array  $selectedIds
-     * @param bool   $withProductCount
-     * @param bool   $includeSub
-     * @param string $relatedEntity
+     * @param array   $categories
```
Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Twig/CategoryExtension.php]: remove space

**Comment by phaseinducer** [comment]
GTM

**Comment by jjanvier** [comment]
GTM

**Pull request merged**